### PR TITLE
Changelog for 0.19.0

### PR DESCRIPTION
**Pull request by angelozerr**
Changelog for 0.19.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+- Qute: add arguments metadata for user-defined tags. See [#928](https://github.com/redhat-developer/quarkus-ls/issues/928).
+- Validate if a BuildItem is `final` or `abstract`. See [#926](https://github.com/redhat-developer/quarkus-ls/issues/926).
+- Support @Route from vertx to display codelens URL. See [#914](https://github.com/redhat-developer/quarkus-ls/issues/914).
+- Support customized template path for @CheckedTemplate. See [#893](https://github.com/redhat-developer/quarkus-ls/issues/893).
+
+### Bug Fixes
+
+- Check if record implement TemplateInstance to provide the support. See [#974](https://github.com/redhat-developer/quarkus-ls/pull/974).
+- Support for Qute arguments. See [#970](https://github.com/redhat-developer/quarkus-ls/issues/970).
+- Qute highlight errors due to formatting . See [#966](https://github.com/redhat-developer/quarkus-ls/issues/966).
```
Comment (datho7561) [inline CHANGELOG.md:19]: I think this entry could be more specific.

eg. "In qute, allow spaces between method parameters and only validate the return type of the method invocation when iterating over a method invocation"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+- fix: allow spaces inside section branch.. See [#965](https://github.com/redhat-developer/quarkus-ls/pull/965).
+- fix: allow inject: with constructor class which have arguments. See [#964](https://github.com/redhat-developer/quarkus-ls/pull/964).
+- Support for inner Enum. See [#963](https://github.com/redhat-developer/quarkus-ls/issues/963).
```
Comment (datho7561) [inline CHANGELOG.md:22]: ... in qute files

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+- Support for void in property part of Qute template. See [#961](https://github.com/redhat-developer/quarkus-ls/issues/961).
+- Wrong syntax HTML when generate by clicking link. See [#955](https://github.com/redhat-developer/quarkus-ls/issues/955).
```
Comment (datho7561) [inline CHANGELOG.md:24]: Template files generated by clicking on a code lens were missing the closing `</html>`.

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## [0.19.0](https://github.com/redhat-developer/quarkus-ls/milestone/21?closed=1) (August 14, 2024)
+
+### Enhancements
+
+- Qute: Cannot locate hyphenated template name. See [#975](https://github.com/redhat-developer/quarkus-ls/issues/975).
```
[the remark this answers is not in the document]
Comment (angelozerr) [reply CHANGELOG.md:7]: Good catch, fixed.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,35 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## [0.19.0](https://github.com/redhat-developer/quarkus-ls/milestone/21?closed=1) (August 14, 2024)
+
+### Enhancements
+
+- Qute: Cannot locate hyphenated template name. See [#975](https://github.com/redhat-developer/quarkus-ls/issues/975).
+- fix: qute(NoMatchingTemplate) if using @CheckedTemplate with basePath. See [#972](https://github.com/redhat-developer/quarkus-ls/pull/972).
```
[the remark this answers is not in the document]
Comment (angelozerr) [reply CHANGELOG.md]: Good catch, fxed

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,35 @@
...
+- fix: separate user tag parameters inlay hint. See [#968](https://github.com/redhat-developer/quarkus-ls/pull/968).
+- Qute: add arguments metadata for user-defined tags. See [#928](https://github.com/redhat-developer/quarkus-ls/issues/928).
+- Validate if a BuildItem is `final` or `abstract`. See [#926](https://github.com/redhat-developer/quarkus-ls/issues/926).
+- Support @Route from vertx to display codelens URL. See [#914](https://github.com/redhat-developer/quarkus-ls/issues/914).
+- Support customized template path for @CheckedTemplate. See [#893](https://github.com/redhat-developer/quarkus-ls/issues/893).
+
+### Bug Fixes
+
+- Check if record implement TemplateInstance to provide the support. See [#974](https://github.com/redhat-developer/quarkus-ls/pull/974).
+- Support for Qute arguments. See [#970](https://github.com/redhat-developer/quarkus-ls/issues/970).
```
[the remark this answers is not in the document]
Comment (angelozerr) [reply CHANGELOG.md]: Good catch, fxed

File: CHANGELOG.md (hunk 1 above)
Comment (angelozerr) [reply CHANGELOG.md]: updated

File: CHANGELOG.md (hunk 2 above)
Comment (angelozerr) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 3 above)
Comment (angelozerr) [reply CHANGELOG.md]: fixed

**Pull request merged**